nf_core/pipeline-template/.github/workflows/download.yml

```diff
@@ -0,0 +1,30 @@
+name: nf-core download
+# This workflow is triggered on pushes to the dev branch, PRs and on release published events.
+# It runs the `nf-core download` command to ensure that the pipeline can be downloaded.
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Nextflow
+        uses: nf-core/setup-nextflow@v1
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.7"
+          architecture: "x64"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install nf-core
+
+      - name: Download the pipeline
+        env:
+          NXF_SINGULARITY_CACHEDIR: ./
+        run: nf-core download {{ name }} -r dev -c singularity --singularity-cache-copy -x tar.bz2 --force {%- endraw %}
```

Comment:
I'm not sure why you need `{%- endraw %}` but I think there is a `{% raw %}` missing.